metadata-io/src/main/java/com/linkedin/metadata/service/UpdateIndicesService.java

```diff
@@ -180,8 +180,12 @@ private void handleUpdateChangeEvent(
       updateSystemMetadata(event.getSystemMetadata(), urn, aspectSpec, aspect);
     }
 
-    // Step 1. Handle StructuredProperties Index Mapping changes
-    updateIndexMappings(urn, entitySpec, aspectSpec, aspect, previousAspect);
+    try {
+      // Step 1. Handle StructuredProperties Index Mapping changes
+      updateIndexMappings(urn, entitySpec, aspectSpec, aspect, previousAspect);
+    } catch (Exception e) {
+      log.error("Issue with updating index mappings for structured property change", e);
```

Comment:
Perhaps log the urn and the aspect?

Reply:
both the urn and aspect should be picked up by the structured logging

Reply:
+1 sgtm